aws/structure_test.go

```diff
@@ -1199,70 +1199,6 @@ func TestFlattenPolicyAttributes(t *testing.T) {
 	}
 }
 
-func TestNormalizeJsonString(t *testing.T) {
-	var err error
-	var actual string
-
-	// Well formatted and valid.
-	validJson := `{
-   "abc": {
-      "def": 123,
-      "xyz": [
-         {
-            "a": "ホリネズミ"
-         },
-         {
-            "b": "1\\n2"
-         }
-      ]
-   }
-}`
-	expected := `{"abc":{"def":123,"xyz":[{"a":"ホリネズミ"},{"b":"1\\n2"}]}}`
-
-	actual, err = normalizeJsonString(validJson)
-	if err != nil {
-		t.Fatalf("Expected not to throw an error while parsing JSON, but got: %s", err)
-	}
-
-	if actual != expected {
-		t.Fatalf("Got:\n\n%s\n\nExpected:\n\n%s\n", actual, expected)
-	}
-
-	// Well formatted but not valid,
-	// missing closing squre bracket.
```

Comment:
Do you mind just PRing this part of the test upstream? https://github.com/hashicorp/terraform/blob/master/helper/structure/normalize_json_test.go

Reply:
Sure, here we go: https://github.com/hashicorp/terraform/pull/17122